lib/torrent.js

```diff
@@ -323,6 +323,13 @@ class Torrent extends EventEmitter {
       })
     }
 
+    // add BEP09 peer-address
+    // TODO: It should only be included if the client can discover its public IP address and determine its reachability.
```

Comment:
What's the downside of not following this line, which I think comes from the spec?

Reply:
I left this as a `TODO` as I don't know how to implement this check 😅
Any help or suggestion here will be more than welcome ❤️

Reply:
Oh, re-reading it again, I think I understand what it means! It's saying that if you're creating a magnet link you should not include your own IP address until you are sure that your IP is public and reachable. For example, if your IP is 192.168.1.100, it's a private IP and there's no point including it.

Since this PR is just about processing these URLs and not generating them, I think we can safely ignore it and remove the TODO.

Reply:
Done! 👍